js/components/util/files.ts

```diff
@@ -255,3 +256,25 @@ export class SmartStorageBroker implements MassStorage<unknown> {
         return null;
     }
 }
+
+/**
+ * Load binary file into memory.
+ *
+ * @param file File object to read into memory
+ * @param address Address at which to start load
+ * @param debug Debugger object
+ * @returns resolves to true if successful
+ */
+
```

Comment:
Should I give up commenting about the extra lines between the docs and the function? :-)